orm_lib/inc/drogon/orm/SqlBinder.h

```diff
@@ -323,7 +323,7 @@ class DROGON_EXPORT SqlBinder : public trantor::NonCopyable
     SqlBinder &operator=(SqlBinder &&that) = delete;
     ~SqlBinder();
     template <typename CallbackType,
-              typename traits = FunctionTraits<CallbackType>>
+              typename traits = FunctionTraits<std::decay_t<CallbackType>>>
```

Comment:
Use `decay<T>::type`. `decay_t` is C++17

Hmm... seems our CI is using C++17 even though I specified c++14 mode. I'll fix it later